task1-gradient-descent/gradient_descent_soner.ipynb

```diff
@@ -0,0 +1,1627 @@
+{
```

Comment:
<p>See this video for the logic behind:</p><p>https://www.youtube.com/watch?v=CbsegCURCd0</p><p> </p><p>The answer is correct but I'm not sure if the gist behind is understood here. This is a good explanation:</p><p>https://stackoverflow.com/questions/49147774/what-is-random-state-in-sklearn-model-selection-train-test-split-example#:~:text=If%20you%20don%27t,and%20test%20datasets.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>